pages/linux/autorandr.md

```diff
@@ -11,7 +11,11 @@
 
 `autorandr`
 
-- Change the profile:
+- Load the first detected profile:
+
+`autorandr -c`
```

Comment:
```suggestion
- Load the first detected profile:

`autorandr --change`
```

Mandatory fix: [long option must be used](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax).